app/jobs/broadcast_survey_stats_job.rb

```diff
@@ -4,6 +4,6 @@ class BroadcastSurveyStatsJob < ApplicationJob
   queue_as :broadcasts
 
   def perform(survey)
-    SurveyResultChannel.broadcast_to(survey, 'message': 'testing')
+    SurveyResultChannel.broadcast_to(survey, AnswerGroupCalculationService.new(survey).calculate.export)
```

Comment:
You could also pass a hash instead of already stringified JSON :)

Reply:
Does not realy end well